apps/aecore/src/aec_chain_state.erl

```diff
@@ -1310,3 +1412,27 @@ fork_result(Count, #{signalling_block_count := SigCount}) when Count >= SigCount
     true;
 fork_result(_Count, _Fork) ->
     false.
+
+setup_etc_cache() ->
+    %% Lager might not be up so don't bother with logging
+    Self = self(),
+    Tab = ?RECENT_CACHE,
+    Keypos = #recent_blocks.key,
+    case ets:info(Tab, name) of
+        undefined ->
+            spawn(fun() ->
```

Comment:
Why timeout?

Reply:
I think this is good practice, even though it is very unlikely to trigger (but not impossible).